html/index.html

```diff
@@ -148,33 +140,36 @@ <h3>Starting with a staircase</h3>
               Haskell. The language soon became a natural 
               part of lessons and meant that it was easy to 
               progress to the next step. </p>
-              <div class="line"></div>
-              <h2><blockquote> CHILDREN COULD CREATIVELY 
-                ENJOY INVENTING FUNCTIONS</blockquote></h2>
-              <h3>Programming</h3>
-              <p>Very soon the children were able to 
-                construct their own rod calculator in 
-                Haskell, by first constructing the engine 
-                using the Colour data type. Then 
-                the children asked the same role-play 
-                questions, this time by typing the language 
-                into the interpreter window. It soon 
-                became apparent that errors in the engine 
-                meant that the interpreter would not 
-                run, and that this would require carefully 
+              <p>run, and that this would require carefully 
                 debugging and retyping. The children 
                 could spot bugs in the engine because they 
                 had already worked out the answer. The 
                 staircase was a familiar pattern, so they 
                 could predict what the output should be. 
-              </p>
-              <p>Children were also able to edit the 
+                Children were also able to edit the 
                 Colour data type to model odd or even 
                 staircases and test their functions to see 
                 what the effect would be on the derived 
                 successor function. A transcript (see 
-                insert) shows a pupil working with the 
-                number names for the rods, measured 
+                insert) shows a pupil working with the </p>
+            </section>
+            <div class="line"></div>
+             <h2 class="quote"><img src="../images/blackdot.png" class="blackdot"><blockquote>
```

Comment:
Could you make normal indents for tags, it's not readable. Each tag on a new line

Reply:
And need to check this whole doc for compliance